pages/en/lb4/Creating-components.md

```diff
@@ -228,6 +230,33 @@ export class AuthenticationProvider {
 }
 ```
 
+## Extends Application
```

Comment:
This section has many grammatical and stylistic errors.  Rather than go through them laboriously here, I'd prefer to either edit the text in thebranch directly or wait til the PR lands and then edit the text (none of the changes would require technical review) in master (ie. gh-pages).  @jannyHou LMK which you'd prefer.

Reply:
hi @crandmck I think now this PR is still waiting for technical review, I will let you know when it's done but before merging.